scripts/ci/gitlab/pipeline/test.yml

```diff
@@ -58,6 +58,18 @@ cargo-fmt:
     - cargo +nightly fmt --all -- --check
     - rusty-cachier cache upload
 
+cargo-fmt-manifest:
+  stage: test
+  extends:
+    - .docker-env
+    - .test-refs
+  script:
+    - cargo install zepter --locked --version 0.10.0 -q -f --no-default-features && zepter --version
+    - zepter format features
+    - echo "👉 Hello developer! If you see this CI check failing then it means that one of the your changes in a Cargo.toml file introduced ill-formatted or unsorted features. Please take a look at 'docs/STYLE_GUIDE.md#manifest-formatting' to find out more. In minor cases, the diff below may help you debug:"
+    - git diff --exit-code
```

Comment:
Relying on `git diff` works, but for CI convenience, could you add a `--check` option to zepter that errors out when it's unhappy about the formatting? (just like `cargo fmt --check`)

Reply:
Yea it does have the option, but it will not print a diff. Anyway i can change it.